### PR TITLE
new blog for agent arena

### PR DESCRIPTION
**Pull request by arthbohra**

### CODE REVIEW

File: assets/img/blog/agent-arena/blog_post_14_arena_demo.gif
Comment (infwinston) [inline assets/img/blog/agent-arena/blog_post_14_arena_demo.gif:1]: remove this file?

File: assets/img/blog/agent-arena/blog_post_14_arena_demo_final.gif
Comment (infwinston) [inline assets/img/blog/agent-arena/blog_post_14_arena_demo_final.gif:1]: remove this file?

File: assets/img/blog/agent-arena/blog_post_14_arena_demo.gif
Comment (arthbohra) [reply assets/img/blog/agent-arena/blog_post_14_arena_demo.gif:1]: Resolved

File: _posts/2024-10-03-agent-arena.md [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+  - name: Kai Wen
+  - name: Sai Kolasani
+  - name: Wei-Lin Chiang
+  - name: Anastasios N. Angelopoulos
+  - name: Joseph Gonzalez
+  - name: Ion Stoica
+  - name: Shishir G. Patil
+---
+
+<iframe width="560" height="315" src="https://youtu.be/qLOsfwrAtqY"
```
Comment (infwinston) [inline _posts/2024-10-03-agent-arena.md:24]: there should be embed

Comment (arthbohra) [reply _posts/2024-10-03-agent-arena.md]: fixed

**Pull request merged**